content/explicit_rma_intro.tex

```diff
@@ -0,0 +1,16 @@
+The explicit non-blocking \ac{RMA} routines described in this section are
+one-sided communication mechanisms similar to the \ac{RMA} routines
+described in Section~\ref{sec:rma}. Most of the semantics for these
+routines are the same as for regular \ac{RMA} routines.
+
+All explicit non-blocking \ac{RMA} routines take a handle to a request
+object as parameter. Request objects are of the type
+\CTYPE{shmem\_request\_t}. The request object is used to track the
+status of \ac{RMA} operations. A request object can be used to track one
+or more operations. If a request object is used for multiple
+communication operations, the resources assigned to the request object
+can be shared.
```

Comment:
If the library decides to not provide resources, it is still correct. IMO, assigning of resources is more of motivation than the requirement.